src/General/@next/Icon/icons/icons.ts

```diff
@@ -0,0 +1,31 @@
+import { SVGProps } from 'react';
+import * as Icons from './../components';
```

Comment:
Hmm, a bit concern about this one, have you verified if the icons tree-shakeable?